src/main/java/org/broadinstitute/hellbender/utils/nio/SeekableByteChannelPrefetcher.java

```diff
@@ -145,6 +149,7 @@ public void close() throws IOException {
     }
 
     public SeekableByteChannelPrefetcher(SeekableByteChannel chan, int bufSize) throws IOException {
+        this.prefetcherIndex = (prefetcherCount++);
```

Comment:
Volatile wouldn't work (it just makes the individual operations atomic, not the read-assign-write as a whole as is needed). We could use atomics, or a lock, but that seems overkill for a debug variable. I don't think it's the end of the world if multiple threads end up with the same name, and I think that's preferable to adding complexity. I can be swayed, though.